release-calendar/src/client/models/view-models.ts

```diff
@@ -56,3 +56,13 @@ export class CurrentReleases {
 
   map: Map<Channel, string>;
 }
+
+export class SingleReleasePromotion {
```

Comment:
What is the point of this class? Can we use `Promotion` since we have it already?